checker/examples/MavenExample/pom.xml

```diff
@@ -15,7 +15,10 @@
     <!-- These properties will be set by the Maven Dependency plugin -->
     <!-- Change to jdk7 if using Java 7 -->
```

Comment:
Here and below, there are instructions regarding what to do if using Java 7.  I suggest that instead we introduce a variable about JDK version.  That way, users can set that variable and change nothing else.  This will be less error-prone.